findmyparadise_frontend/src/components/Guests.vue

```diff
@@ -1,13 +1,25 @@
 <template>
   <div>
     <h5>
-      Adults: <button type="button" id="DecresaeAdults" class="btn" @click="DecreaseAdults()">-</button> {{AdultsNum}} <button type="button" class="btn" @click="IncreaseAdults()">+</button>
+      Adults:
+      <button
+        type="button"
+        id="DecresaeAdults"
+        class="btn"
+        @click="DecreaseAdults()"
```

Comment:
Let's all agree to camel case methods and variables. decreaseAdults(), increaseAdults(), adultsNum, etc